types/node/test/crypto.ts

```diff
@@ -1377,42 +1377,23 @@ import { promisify } from 'node:util';
     new crypto.webcrypto.CryptoKey(); // Illegal constructor
 
     crypto.webcrypto.subtle.generateKey({ name: 'HMAC', hash: 'SHA-1' }, true, ['sign', 'decrypt', 'deriveBits']).then((out) => {
-        if (out instanceof crypto.webcrypto.CryptoKey) {
-            out.algorithm; // $ExpectType KeyGenParams
-            out.extractable; // $ExpectType true
-        } else {
-            out.publicKey.type; // $ExpectType "public"
-            out.privateKey.type; // $ExpectType "private"
-            out.publicKey.extractable; // $ExpectType boolean
-            out.privateKey.extractable; // $ExpectType true
-        }
+        out.algorithm; // $ExpectType KeyAlgorithm
+        out.extractable; // $ExpectType boolean
     });
 }
 
 {
     // Note: The following tests are not examples of correct usage of these APIs and are simply for typechecking testing.
     const subtle = crypto.subtle;
-    const rsaAes: crypto.webcrypto.SubtleCrypto.RsaOaepAesParams = { name: 'RSA-OAEP' };
-    const deriveAlgo: crypto.webcrypto.SubtleCrypto.DeriveAlgorithm = {
-        name: 'PBKDF2',
-        iterations: 1000,
-        hash: 'SHA-1',
-        salt: new Uint8Array([1, 2, 3, 4])
-    };
     // The lack of top level await makes it annoying to use generateKey so let's just fake it for typings.
     const key = null as unknown as crypto.webcrypto.CryptoKey;
     const buf = new Uint8Array(16);
 
-    subtle.decrypt(rsaAes, key, Buffer.alloc(8)); // $ExpectType Promise<ArrayBuffer>
```

Comment:
Why were the tests for those methods deleted? Doesn't seem ideal

Reply:
I just went after deleting variables that used removed types.

I don't see why more tests couldn't get added referencing the standard dictionaries.

Reply:
Feel free to pick up on that.

Reply:
Thought so, just wanted to be sure, I'll add them back appropriately afterwards then